ughubHelp.py

```diff
@@ -69,33 +84,85 @@ def PrintCommands():
 	for cmd in GetCommandsInHelp():
 		print("  {0}".format(cmd))
 
+def PrintCommandNames():
+	result = ""
+	for cmd in GetCommandsInHelp():		
+		for c in cmd.split(","):
+			result += c.strip() + "\n"			
+	print(result[:-1], end ="")
+
 # Prints help for the command specified in 'cmd'.
-def PrintCommandHelp(cmdName):
+def PrintCommandHelp(cmdName, args=[]):
+
+	shortdesc = ughubUtil.HasCommandlineOption(args, ("--shortdescription",))
```

Comment:
Thinking about it... maybe `--short` would suffice? I think `ughub help list --short` is probably expressive enough. This only concerns the command line argument, not the internal variables and fields - here `shortdescription` / `shortdesc` is better, I think.